packages/botfuel-dialog/src/config.js

```diff
@@ -135,10 +133,7 @@ const getComponentRoots = function (config): string[] {
  */
 const getConfiguration = (botConfig: RawConfig = {}): Config => {
   // get the config by extending defaultConfig with botConfig
-  const config = _.merge(
-    defaultConfig,
-    _.omitBy(botConfig, (val, key: string): boolean => !whitelist.includes(key)),
-  );
+  const config = _.merge(defaultConfig, botConfig);
```

Comment:
If we are now allowing any keys in the configuration file the `config.custom` key become useless in Config type